config/routes.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  #!original should uncomment the links below...
+  # resources :users, only: [:show]
```

Comment:
- [ ] @admirerbrown it's not good practice to have commented code in your codebase, kindly remove all the commented code.